src/linear_algebra/basis.lean

```diff
@@ -1006,18 +1021,6 @@ begin
 end
 
 open fintype
-variables (h : is_basis α v)
-
-local attribute [instance] submodule.module
-
-noncomputable def equiv_fun_basis [fintype ι] : β ≃ (ι → α) :=
-calc β ≃ (ι →₀ α) : (module_equiv_finsupp h).to_equiv
-   ... ≃ (ι → α)                              : finsupp.equiv_fun_on_fintype
-
-theorem vector_space.card_fintype [fintype ι] [fintype α] [fintype β] :
-  card β = (card α) ^ (card ι) :=
-calc card β = card (ι → α)    : card_congr (equiv_fun_basis h)
-        ... = card α ^ card ι : card_fun
 
 theorem vector_space.card_fintype' [fintype α] [fintype β] :
```

Comment:
```suggestion
theorem vector_space.card_fintype [fintype α] [fintype β] :
```
Since the old one moved to the `module` namespace, we should be able to remove the `'` here.